src/utils.py

```diff
@@ -46,6 +52,12 @@ def create_pull_request(
             )
             return response_data["number"]
 
+        elif (
+            response.status_code == 422
+            and "A pull request already exists for" in response.text
+        ):
+            raise PullRequestExistsError
```

Comment:
Rather than raising an error we can just add a warning that pull request exists and return `None`.

Reply:
Done.